.github/workflows/http3-linux.yml

```diff
@@ -98,7 +98,7 @@ jobs:
           cache-name: cache-quictls-no-deprecated
         with:
           path: /home/runner/quictls/build
-          key: ${{ runner.os }}-http3-build-${{ env.cache-name }}-${{ env.quictls-version }}
+          key: ${{ runner.os }}-http3-build-${{ env.cache-name }}-${{ env.quictls-version }}-quic1
```

Comment:
Adding these in because otherwise we hit weird behaviour in the `cache` logic where it both restores the cache, and tries to generate the cache (I think because `steps.cache-quictls-no-deprecated.outputs.cache-hit` is equal to `false`, not the empty string, because it's restoring from a different cache that sort-of matches the requested one)